packages/ui-components/jest.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
The jest config is significantly different from all other projects e.g. `fe-fpm-writer/jest.config.js`. In some cases, it would make sense to move some of the settings to the root `tsconfig` e.g. adding tsx to `collectCoverageFrom` and other cases I'd like to understand why the settings are needed.

Reply:
I have adapted jest-config based on the other config file. Could you review it again. tsx file added to collect to get the coverage for react components test files.

Reply:
I have adapted jest-config based on the other config file. Could you review it again. tsx file added to collect to get the coverage for react components test files.